services/backend/api_doc/strategic_ecosystems.js

```diff
@@ -11,30 +11,3 @@
  *    "national_percentage": 0.1523
  *  }
  */
-
```

Comment:
delete this file too, the endpoints that use `SEInGeofenceDetailExample` aren't being used by the front